ROOTFS/media/rpi/sage-utils/dhcp-pxe/nfs/etc/rc.local

```diff
@@ -38,12 +38,23 @@ else
   echo "Error: SD CARD not found will continue without SD Scratch Space" | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
 fi
 
+echo "Initialzing DNS Nameservers" | xargs -L 1 echo `date +'[%Y-%m-%d %H:%M:%S]'` >> /etc/rc.local.logs
+rm /etc/resolv.conf
+printf "nameserver 8.8.8.8\nnameserver 8.8.4.4" > /etc/resolv.conf
```

Comment:
why can't this be in the file `ROOTFS/media/rpi/sage-utils/dhcp-pxe/nfs/etc/resolv.conf` instead of being set at each boot?

Reply:
resolved, after discussion on slack. Please confirm

Reply:
for memory (removed symlink included in Ubuntu 20 to /etc/resolv.conf then copied into ROOTFS/media/rpi/sage-utils/dhcp-pxe/nfs/etc/resolv.conf)